### DISCUSSION TITLE
How do I enable the kernel mode driver?

### DISCUSSION THREAD

**Discussion by ZGuardianTV** [winsiderss/systeminformer#2029] [Questions / Answers] [answered]
How do I enable the KSystemInformer kernel mode driver? I tried searching for it all I found was this https://github.com/winsiderss/systeminformer/discussions/1375 but it didn't include any information on how to actually enable the driver

**Comment by jxy-s** [comment, marked as answer]
System > Options... > General > Enable kernel-mode driver
